packages/api-util/src/__tests__/api-util.test.tsx

```diff
@@ -5,7 +5,8 @@ import {
   validationErrorsAreSupported,
   clearAjvErrorForPath,
 } from '../api-util';
-import { CARD_VIEW_PAGE_SIZE } from '../hooks';
+
+const testUrl = new URL('test', `https://test.com`);
```

Comment:
You should use example.com mainly because thats what its for. test.com is some random website. Not sure about constructing the url each time but also I don't have an easy way to deal with it